### ISSUE TITLE
Proposal: Change use of HexEncoder to bytes.hex

### ISSUE THREAD

**Issue by AnonymousDapper** [AnonymousDapper/aiotnb#13]
In situations where we have a `VerifyKey.encode(encoder=HexEncoder)`, it may be a good idea for code readability to replace that with `bytes(VerifyKey).hex()`.

I want your opinions on this, please.

**Comment by AbhayAysola** [comment]
Yeah I think this would be better in terms of readability, but we would need to test it to make sure conversions are happening correctly.

**Comment by AnonymousDapper** [comment]
> Yeah I think this would be better in terms of readability, but we would need to test it to make sure conversions are happening correctly.

As far as I can tell, after looking around a bit, that's all HexEncoder does under the hood. We should be good unless some crazy magic happening that I missed.

**Comment by AbhayAysola** [comment]
Yeah that's great, I think we can switch over.

**Comment by Aareon** [comment]
LGTM
